### PR TITLE
improved inventory navigation

### PR DESCRIPTION
**Pull request by Infernum1**
add "skip to last" and "back to first" button to easily skip to last page and vice-versa

### CODE REVIEW

File: bot/cogs/core/paginator.py [hunk 1]
```diff
@@ -82,6 +84,11 @@ async def paginate_embed(
             elif emoji == RIGHT_ARROW:
                 if page < page_count - 1:
                     page += 1
+            elif emoji == SKIP_TO_LAST:
+                page += page_count - 1
```
Comment (Iapetus-11) [inline bot/cogs/core/paginator.py]: Shouldn't this be `page = page_count - 1`

**Review by Iapetus-11 — CHANGES_REQUESTED** [review]

File: bot/cogs/core/paginator.py (hunk 1 above)
Comment (Infernum1) [reply bot/cogs/core/paginator.py]: well it works with both `page = page_count - 1` and `page += page_count - 1`, I can make changes if you want the former

edit: I think it only works with the latter if the page count is 0 though... so I will make the edit

Comment (Iapetus-11) [reply bot/cogs/core/paginator.py]: > I think it only works with the latter if the page count is 0 though... so I will make the edit

Exactly

Comment (Iapetus-11) [reply bot/cogs/core/paginator.py]: https://github.com/Iapetus-11/Villager-Bot/pull/272/commits/8495b94bd486adf8c165693c59fcac5778ab34b0

**Review by Iapetus-11 — APPROVED** [review]

**Pull request merged**